lib/ruby-mws/api/feed.rb

```diff
@@ -43,6 +46,28 @@ def submit_feed(type=nil, content_params={})
       end
 
       private
+
+      def content_for_product_list_price(opts={})
+        Nokogiri::XML::Builder.new do |xml|
+          xml.AmazonEnvelope("xmlns:xsi" => "http://www.w3.org/2001/XMLSchema-instance",
+                             "xsi:noNamespaceSchemaLocation" => "amzn-envelope.xsd") { # add attrs here
+            xml.Header {
+              xml.DocumentVersion "1.01"
+              xml.MerchantIdentifier @connection.seller_id
+            }
```

Comment:
It looks like we're using this header a lot in this file. Maybe it's time to abstract it out? Maybe something that can be used like:

```ruby
amazon_envelope_with_header do |xml|
  xml.MessageType "Price"
  ...
end
```